prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -820,8 +820,15 @@ public async Task<ActionResult> ApproveSite(int siteId)
             }
 
             var updatedSite = await _siteService.ApproveSite(siteId);
-            await _emailService.SendSiteApprovedPharmaNetAdministratorAsync(site);
-            await _emailService.SendSiteApprovedSigningAuthorityAsync(site);
+            if (site.ActiveBeforeRegistration)
+            {
+                await _emailService.SendSiteActiveBeforeRegistrationAsync(site.Id, site.Organization.SigningAuthority.Email);
+            }
+            else
+            {
+                await _emailService.SendSiteApprovedPharmaNetAdministratorAsync(site);
```

Comment:
is the intention that the pharmanet administrator email is not sent at all for these sites?

Reply:
Sarah just confirmed, we are not sending to the Pharmanet Admin.